contrib/azure/azure-coreos-cluster

```diff
@@ -85,6 +88,11 @@ if not args.ssh_cert and not args.ssh_thumb:
       args.ssh_cert = './ssh-cert.cer'
   print 'Generated SSH certificate with thumbprint ' + args.ssh_thumb
 
+# generate coreos discovery url
+if not args.no_discovery_url:
+  print 'Generating new CoreOS discovery URL for the cluster'
+  subprocess.call(['bash', '-c', 'python ./create-azure-user-data $(curl -s https://discovery.etcd.io/new)'])
```

Comment:
Not really IMO, the script won't work today if you try to run it outside of `contrib/azure`, first thing it'll fail on is the call to `openssl` to create the ssh certificates. If you specify one on the command line it should work but by default (what's on the docs) is that it'll create a new ssh cert for the cluster. The doc also says "change directory into contrib/azure while following this documentation."